packages/contracts/test/plugin/plugin-setup-processor.ts

```diff
@@ -697,13 +817,55 @@ describe('Plugin Setup Processor', function () {
       );
     });
 
-    describe('PrepareUpdate', function () {
+    describe('isValidUpdate', function () {
+      it('accepts major updates', async () => {
+        expect(await psp.isValidUpdate([1, 0, 0], [2, 0, 0])).to.equal(true);
```

Comment:
Don't we need a test for a larger jump ? like: `.....isValidUpdate([1, 0, 0], [4, 5, 0])`

Reply:
I think we have it covered now